src/lib/airlock/airlock.js

```diff
@@ -0,0 +1,56 @@
+import { base } from '../airtable/airtable';
+import {
+  refreshUserData,
+  clearUserData,
+} from '../redux/userData';
+
+const AUTHENTICATION_ERR_STRING = 'AUTHENTICATION_REQUIRED';
+
+const signupUser = async (email, password, fullname, role) => {
+  try {
```

Comment:
Don't we want to call refreshUserData to persist user data (and log them in for a session) when they sign up?